linkerd/app/outbound/src/ingress.rs

```diff
@@ -133,6 +134,7 @@ impl Outbound<()> {
             .check_new_service::<tcp::Accept, transport::metrics::SensorIo<I>>()
             .push(self.runtime.metrics.transport.layer_accept())
             .push_map_target(tcp::Accept::from)
```

Comment:
i think this should have been removed --- it still compiles because it's now just doing the identity conversion (`tcp::Accept` -> `tcp::Accept`):
```suggestion
```

Reply:
whoops. good catch.